tests/unit/test_cube.py

```diff
@@ -0,0 +1,21 @@
+# encoding: utf-8
+
+"""Unit test suite for `cr.cube.cube` module."""
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import pytest
+
+from cr.cube.cube import Cube
+
+
+class DescribeBaseCrunchCube(object):
```

Comment:
The name should mirror the class under test. So this one would be named `DescribeCube`. In general, a test class tests the methods of a single production class.